src/main/java/gov/usgs/locator/LocInput.java

```diff
@@ -218,6 +225,7 @@ public boolean readHydra(String fileString) {
     BayesianSpread = scan.nextDouble();
     scan.next().charAt(0); // rstt (not used)
     UseSVD = !LocUtil.getBoolean(scan.next().charAt(0)); // True when noSvd is false
+    ReassessInitialPhaseIDs = true;

```

Comment:
why is this hard coded to true here? I guess a default value because it was not provided in the string?

Reply:
Yes, the legacy hydra locator format does not include this parameter